falcon/util/uri.py

```diff
@@ -49,9 +49,9 @@
 _HEX_DIGITS = '0123456789ABCDEFabcdef'
 
 # This map construction is based on urllib's implementation
-_HEX_TO_BYTE = dict(((a + b).encode(), bytes([int(a + b, 16)]))
+_HEX_TO_BYTE = {(a + b).encode(): bytes([int(a + b, 16)])
```

Comment:
I guess this change we could actually keep. The old form probably could not use dict comprehension because it had to support Python 2.6.